core/src/test/scala/io/snappydata/SnappyFunSuite.scala

```diff
@@ -136,6 +161,7 @@ abstract class SnappyFunSuite
     // GemFireXD stop for local mode is now done by SnappyContext.stop()
     println(" Stopping spark context = " + SnappyContext.globalSparkContext)
     SnappyContext.stop()
```

Comment:
We should really get rid of this. I had suggested it to Namrata where she had changed snappy cleanup to happen in SparkContext stop via listener bus. Its unnecessary and confusing to users because it is exactly equivalent to SparkContext stop.